src/containers/walletContainer.js

```diff
@@ -149,6 +149,10 @@ class WalletContainer extends Component {
                 open: false
             })
         }
+        this.setState({
```

Comment:
If Amount is equal to zero (0), we should give warning to user.

Reply:
ok i did modify